src/rustdoc_schema.graphql

```diff
@@ -878,14 +916,36 @@ type AssociatedConstant implements Item {
 
   # properties for AssociatedConstant
   """
+  A type represented in the "raw" rustdoc JSON representation (see `interface RawType`)
+  Currently serialized as a serde_json String instead of a proper RawType
+  ```rust
+  const fn five() -> i32 { 5 };
+  struct Years(i32);
+  trait BatchIterator<const MIN: usize> {	    // rustdocs type field
+    const MIN     : usize        = 16       ; // {"primitive":"usize"}
+    const MIN_SIZE: usize        = MIN      ; // {"primitive":"usize"}
+    const LOG_AS  : &'static str = "batch"  ; // {"borrowed_ref":{"lifetime":null,"mutable":false,"type":{"primitive":"str"}}}
+    const YEAR    : Years        = Years(42); // {"resolved_path":{"name":"Years","id":"0:3:1633","args":{"angle_bracketed":{"args":[],"bindings":[]}}}}
+    const EXPR2_2 : i32          = 2+2      ; // {"primitive":"i32"}
+    const FN_FIVE : i32          = five()   ; // {"primitive":"i32"}
+  }
+  ```
+  """
+  type_: String
```

Comment:
I don't think adding `type_` as a property in either const or associated const is a good idea. It isn't in line with the rest of the crate's approach, and it prevents us from doing something better later.

I'd prefer to not add it. This shouldn't limit your ability to extract that information via the ID + index lookup method I mentioned.

Reply:
Yeah, that was before I've reviewed your ID suggestion, removed types